ga4gh/datamodel/reads.py

```diff
@@ -68,6 +68,8 @@ class SamFlags(object):
     """
     NUMBER_READS = 0x1
     PROPER_PLACEMENT = 0x2
+    UNMAPPED = 0x4
+    NEXT_SEGMENT_UNMAPPED = 0x8
```

Comment:
Yeah, that's a mistake in that guide. It should be checking:
`read.alignment` and `read.nextMatePosition`

I'll fix that, thanks.